doc/about.rst

```diff
@@ -22,11 +22,22 @@ Governance
 The decision making process and governance structure of scikit-learn is laid
 out in the :ref:`governance document <governance>`.
 
-Authors
--------
+.. _authors:
```

Comment:
Should this tag be adapted? Shall we also credit the original academical authors of scikit-learn and its associated publication?

Reply:
We can adapt this tag if you think that it is important. I did not think it was, as it's purely an internal tag, and I was worried of doing my git grep wrong and thus missing some places pointing there.

I'm not in favor of crediting the original academic authors and the associated publication more than they already are: this list reflect the people involved in the first couple of years, which is a small fraction of the total road that got scikit-learn to where it is today.

Reply:
Actually, regarding the tags, I added it to make sure that old html links that point to the anchor "#authors" in html still worked.

I'm relearning an old lesson: each time one does a non obvious thing, it should come with a comment :)